Android/Shared/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="alert_message_webview_not_found">Chromium WebView package required to run the app.</string>
+    <string name="alert_title_error">Error</string>
+    <string name="ok">OK</string>
+    <string name="alert_message_only_bim_files">Only .bim files can be opened.</string>
```

Comment:
With the new contract, is this string still used?

Reply:
Yes, the user can still select any file and will be shown an alert if they choose a file without a .bim extension.